db/sample_data/open_studios.rb

```diff
@@ -12,5 +12,8 @@ def ensure_open_studios_event(start_date)
 OpenStudiosEvent.all.each do |os_event|
   Artist.active.each do |artist|
     artist.open_studios_events << os_event
+  rescue ActiveRecord::RecordInvalid => e
```

Comment:
this was an incidental fix i put in to get my dev setup working on a different machine.  `sample data` should be pretty easy and bullet proof (imo)